prov/shm/src/smr_ep.c

```diff
@@ -1649,10 +1255,11 @@ int smr_srx_context(struct fid_domain *domain, struct fi_rx_attr *attr,
 
 	if (attr->op_flags & FI_PEER) {
 		smr_domain->srx = ((struct fi_peer_srx_context *) (context))->srx;
-		smr_domain->srx->peer_ops = &smr_srx_peer_ops;
+		//smr_domain->srx->peer_ops = &smr_srx_peer_ops;
```

Comment:
Is it commented intentionally?

Reply:
Oh whoops. Should have just been deleted. Thanks for catching!

Reply:
May I know why it is deleted? Is it moved somewhere else?

Reply:
Oh, yes. The detection of an imported srx is now happening when the EP gets enabled [here](https://github.com/aingerson/libfabric/blob/peer/prov/shm/src/smr_ep.c#L1322)

This is just a draft proposal so a lot of it will probably change when I update :)